public/editor-client/src/api/index.ts

```diff
@@ -1206,3 +1214,60 @@ export const updateGlobalBlocks = async (
 };
 
 //#endregion
+
+//#region Default Templates
+
+export const getPopups = async (
+  url: string
+): Promise<{
+  blocks: APIPopup[];
+  categories: { slug: string; title: string }[];
+}> => {
+  const response = await request(url, {
+    method: "GET"
+  });
+
+  if (response.ok) {
+    const res = await response.json();
+
+    if (isT(res) && isPopupsResponse(res)) {
+      return { blocks: res.collections, categories: res.categories };
+    }
+  }
+
+  throw new Error(t("Failed to load popups"));
+};
+
+export const getPopupData = async (
+  url: string,
+  id: string
+): Promise<DefaultBlockWithID> => {
+  const fullUrl = makeUrl(url, {
+    project_id: id as string
+  });
+
+  const response = await request(fullUrl, {
+    method: "GET"
+  });
+
+  if (response.ok) {
+    const res = await response.json();
+
+    const parsedResult = mPipe(
+      pass(isPopupDataResult),
+      (res) => res.pop()?.pageData,
+      (r) => JSON.parse(r),
```

Comment:
use `Json.parse` from "@brizy/readers"

Reply:
we don't have parse 

![image](https://github.com/ThemeFuse/Brizy/assets/17808787/af73f9c6-828b-47c3-9bd1-a4c65d5c8065)

Reply:
1. yes, but we have `Json.read`, so use it
2. `JSON.parse` returns `any`, that's why you don't get error